### PR TITLE
Reset version page state when currently selected version changes

### PR DESCRIPTION
**Pull request by John-Paul-R**

### CODE REVIEW

**Comment by John-Paul-R** [comment]
Relevant thread: https://discord.com/channels/734077874708938864/980294666794704968

**Review by wafflecoffee — APPROVED** [review]
Urgent fix, will wait for another team review before merging though

**Pull request merged**
